sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_models.py

```diff
@@ -1164,8 +1164,49 @@ def __repr__(self):
             .format(self.positive, self.neutral, self.negative)[:1024]
 
 
-class EntitiesRecognitionTask(DictMixin):
-    """EntitiesRecognitionTask encapsulates the parameters for starting a long-running Entities Recognition operation.
+class AnalyzeBatchActionsType(str, Enum):
+    """The type of batch action that was applied to the documents
+    """
+    RECOGNIZE_ENTITIES = "recognize_entities"  #: Entities Recognition action.
+    RECOGNIZE_PII_ENTITIES = "recognize_pii_entities"  #: PII Entities Recognition action.
+    EXTRACT_KEY_PHRASES = "extract_key_phrases"  #: Key Phrase Extraction action.
+
+class AnalyzeBatchActionsResult(DictMixin):
+    """RecognizeEntitiesActionResult contains the results of a recognize entities action
+    on a list of documents. Returned by `begin_analyze_batch_actions`
+
+    :ivar document_results: A list of objects containing results for all Entity Recognition actions
+        included in the analysis.
+    :vartype document_results: list[~azure.ai.textanalytics.RecognizeEntitiesResult]
+    :ivar bool is_error: Boolean check for error item when iterating over list of
```

Comment:
what does `Always False for an instance of a AnalyzeBatchActionsResult.` mean?

Reply:
we already talked offline, but adding it to this pr for posterity

for a good result we have `is_error` always set to `True`. While for the `AnalyzeBatchActionsError`, we have it as `False` always. This way, when users are iterating through the heterogeneous action results list, they can call `is_error` on each result to see what's an error and what's not

Reply:
Python ❤️